src/core/mcis/cluster.go

```diff
@@ -1537,6 +1551,160 @@ func DeleteAllCluster(nsId string, subString string, forceFlag string) (common.I
 	return deletedClusters, nil
 }
 
+// UpgradeCluster upgrades the TB Cluster object
```

Comment:
```suggestion
// UpgradeCluster upgrades an existing cluster to the specified version
```
조금 더 TB 시스템 관점에서 명확하게 주석을 남기면 좋을 것 같습니다. (클러스터 오브젝트 자체만 업그레이드 하는 것이 아니라, 실제 버전을 업그레이드 하는 것이므로)

Reply:
보완하였습니다.